react/index.js

```diff
@@ -99,7 +99,7 @@ class CategoryMenu extends Component {
           }} />}
           {departments.map(category => (
             <div key={category.id} className="flex items-center">
-              <CategoryItem category={category} subcategoryLevels={+showSubcategories} />
+              <CategoryItem category={category} subcategoryLevels={1 + showSubcategories} />
```

Comment:
What is the semantic of this modification?

Assign it to variable that makes what is being done more clear